utils/activity.ts

```diff
@@ -41,3 +54,9 @@ export const success = () =>
   ActivityResultSuccess.encode({
     kind: "SUCCESS"
   });
+
+export const successWithValue = (value: SuccessValue) =>
+  ActivityResultSuccessWithValue.encode({
+    kind: "SUCCESS_VAL",
+    value
+  });
```

Comment:
While moving into `StoreEycaExpirationActivity`we can maintain `SUCCESS` as kind instead of `SUCCESS_VAL`

Reply:
How can I put an elemnt of type `SuccessValue` if I move the definition outside?

Reply:
You can move `SuccessValue` definition as well :)

Reply:
I know, I didn't undestood to also move `successWithValue`.  My idea was to do a generic implementation, actually, since it sometimes happens to need a return value other than a result info.

Reply:
ditto

Reply:
Done in [#9d07ba5](https://github.com/pagopa/io-functions-cgn/pull/27/commits/9d07ba561352276df16d81ca4859d3d33e9c88fa)